.bazelrc

```diff
@@ -283,6 +286,9 @@ build:linux --copt="-fdata-sections" --host_copt="-fdata-sections"
 # On Linux, use clang lld.
 build:linux --linkopt="-fuse-ld=lld"
 
+# Disable LLVM 20 warning about unused cli argument "-c".
+build:linux --copt='-Wno-unused-command-line-argument' --host_copt='-Wno-unused-command-line-argument'
```

Comment:
Based on CI failing, I think you need -Wno-unknown-warning-option to support versions older than 20 still.